docs/examples/check-email-for-intent.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: default
+title: Check email for Intent
+nav_order: 12
+parent: Examples
+---
+
+# Introduction
+We're going to be creating two workflows one will send an email with a cloud event to trigger the second workflow. The second workflow will read the contents of the email and use AI to work out its intent and respond back if it is negative.
```

Comment:
In this example, we will create two workflows; one will send and email and generate a cloud event, and the other will trigger upon receiving the cloud event, check the contents of an email, use AI to discern the 'intent' of the message, and respond if the intent is deemed to be negative.